test/sandbox/app/components/inherited_from_uncompilable_component.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class InheritedFromUncompilableComponent < UncompilableComponent
+  def call(*)
```

Comment:
I don't believe `call` ever accepts any arguments.

```suggestion
  def call
```